bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/sign-ed25519.ts

```diff
@@ -62,11 +62,13 @@ async function run() {
             bip44Chain,
         );
         console.log(
-            `Public key: ${ed25519Signature.publicKey.publicKey}\nSignature: ${ed25519Signature.signature}`,
+            `Public key: ${ed25519Signature.ed25519PublicKey()}\nSignature: ${
```

Comment:
I feel like just `publicKey` is fine, since this is a `ed25519Signature`

Reply:
I think Thoralf already commented somewhere that `publicKey` is already taken by the property.

Reply:
You know what, nvm